backend/api/system/general.py

```diff
@@ -227,3 +235,40 @@ def post(self):
                 "Error": "Unable to fetch application keys",
                 "SubCode": "InternalServerError",
             }, 500
+
+
+class SystemReleaseAPI(Resource):
+    def post(self):
+        """
+        Fetch latest release version form github and save to database.
+        ---
+        tags:
+          - system
+        produces:
+          - application/json
+        responses:
+          201:
+            description: Saved version successfully to database
+          502:
+            description: Couldn't fetch latest release from github
+          500:
+            description: Internal server error
+        """
+        response = requests.get(
+            "https://api.github.com/repos/hotosm/tasking-manager/releases/latest"
+        )
+        try:
+            tag_name = response.json()["tag_name"]
+            published_date = response.json()["published_at"]
+            release = ReleaseVersion.get()
+            if release is None:
+                release = ReleaseVersion()
+            release.tag_name = tag_name
+            release.published_at = published_date
+            release.save()
```

Comment:
If the release hasn't changed when the request is made (say, by someone trying to call the function outside of the GH action), we should exit out rather than save over.

Reply:
Sounds good